developer-docs/README.md

```diff
@@ -64,102 +64,207 @@ SEMATIC_API_ADDRESS: https://<my_prod1_host:port>
 SEMATIC_API_KEY: <my_prod1_api_key>
 ```
 
-## Releasing
-
-**Note:** Actually pushing the released wheel can only be done if you have
-access to the PyPi repo, which is limited to employees of Sematic.
+## Building the Wheel
 
-We cut releases from the `main` branch, following these steps:
+If you want to build the wheel from source, do:
 
-- Bump the version in `wheel_constants.bzl`, `sematic/versions.py`,
-  and `helm/sematic-server/Chart.yaml`
-- Increment the minor version of the `version` field in
-  `helm/sematic-server/Chart.yaml`. Note that this will be DIFFERENT
-  from the Sematic version you changed in prior steps (chart version
-  will be 1.X.Y, Sematic version is 0.M.N).
-- Update `changelog.md` with the new version number and any missing change
-  entries
-- Make the bump commit
-- Build the UI:
 ```bash
 $ make ui
-```
-- Build the wheel:
-```bash
 $ make wheel
 ```
-- copy wheel from `bazel-bin/sematic/sematic-....whl` into a scratch directory,
-and use a virtual env to test:
-
-```bash
-$ pip3 install <wheel path>
-$ sematic stop
-$ sematic start
-$ sematic run examples/mnist/pytorch
-```
-
-Do this for all supported versions of Python. You can check your virtual env
-Python version using `sematic version` (as well as the Server and Client
-version).
-
-At this point, you should also deploy the Docker image to a cloud Dev environment
-using the [internal Helm charts](/helm/sematic-server), and execute at least one
-pipeline in detached cloud mode. To build the image, use the same process
-you do when usually deploying dev code to a dev env. You can use
-[`serve-dev`](https://github.com/sematic-ai/infrastructure/tree/main/bin)
-to build and deploy the image while on the release branch to deploy the
-release candidate. You will likely also want to smoke test new features
-that were included in the release.
-
-If everything works fine, we are ready to push the release.
-
-```bash
-$ make test-release
-$ make release
-```
 
-Once you have pushed it to PyPi, add the git tag.
-
-```bash
-$ export RELEASE_VERSION=v$(python3 ./sematic/versions.py)
-$ git tag $RELEASE_VERSION
-$ git push origin $RELEASE_VERSION
-```
-
-Next, build and push the server image. Use the dockerfile at
-`docker/Dockerfile.server`.
+## Releasing
 
-```bash
-$ TAG=v$(python3 sematic/versions.py) make release-server
-```
+**Note:** Actually pushing the released wheel can only be done if you have
+access to the PyPI repo, which is limited to employees of Sematic.
```

Comment:
And that non-employees of Sematic won't have access to the `infrastructure` repository either. (So don't try)

Reply:
Done.